site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,173 @@
+
+When [Rodrigo](https://github.com/ReinUsesLisp) improved yuzu’s ASTC decoding, he also added a rule to use native 
+hardware decoding whenever possible. The Nvidia driver tells yuzu it supports ASTC decoding, but as it turns out, they actually use an internal software decoder that is much slower than our own implementation. [Ignoring the Nvidia driver-level software decoder](https://github.com/yuzu-emu/yuzu/pull/4014) produced a massive 
+performance improvement when facing the dreaded ASTC texture format in games. It will still be immediate with Intel GPUs,
+as no software optimizations will beat a dedicated hardware decoder.
+
+## Bug fixes and improvements
+
+Vulkan development is an ongoing process in yuzu, and it has stability problems as expected of a relatively new and complex feature. 
+[Blinkhawk](https://github.com/FernandoS27) made a couple of [critical changes to Vulkan and Asynchronous GPU](https://github.com/yuzu-emu/yuzu/pull/3905), improving stability 
+considerably.
+
+Speaking of Vulkan, many 2D games had their sprites flipped or completely wrong, and once again, we currently lack the 
+extension required to fix this. Therefore, [Rodrigo](https://github.com/ReinUsesLisp) implemented [support for `NV_viewport_swizzle`](https://github.com/yuzu-emu/yuzu/pull/3885).
+This Nvidia-exclusive extension is the only way to solve this problem in a clean manner for now, but a universal method is being 
+developed.
+
+{{< single-title-imgs
+    "Quack (Duck Game)"
+    "./10.png"
+    "./11.png"
+  >}}
+
+The updated libraries (that the migration to Conan brought us) also gave us a new version of the [cubeb](https://github.com/kinetiknz/cubeb) audio engine which adds 
+support for 6 channel audio, allowing [ogniK](https://github.com/ogniK5377) to add [support for surround sound](https://github.com/yuzu-emu/yuzu/pull/3827).
+
+Our good shark, [ogniK](https://github.com/ogniK5377), also fixed [keyboard emulation support](https://github.com/yuzu-emu/yuzu/pull/3926), so expect compatible games to have proper direct input from your keyboards now.
+
+[Morph](https://github.com/Morph1984) implemented the missing support for [`R8G8UI` textures](https://github.com/yuzu-emu/yuzu/pull/3839), fixing both the performance problems and saving crashes that `The Walking Dead` games experienced.
+
+{{< single-title-imgs
+    "Thank you Toxa for the screenshot (The Walking Dead: The Final Season)"
+    "./08.png"
+  >}}
+
+Although objectively a small issue, the mouse cursor didn't hide when running yuzu in full screen, causing a subjectively significant annoyance. Thankfully, [Tobi](https://github.com/FearlessTobi) implemented an [option to automatically hide the mouse](https://github.com/yuzu-emu/yuzu/pull/3892) once it has been inactive after some time.
+
+## An elegant feature of a more civilized age
+
+Recently released in the Early Access build, and coming soon to Mainline, is support for `assembly shaders` (`GLASM`), usually 
```

Comment:
They were merged today

Reply:
Thanks!

Reply:
It still is an extra day until it is in mainline.